analysis/src/Commands.ml

```diff
@@ -176,6 +176,78 @@ let documentSymbol ~path =
     in
     print_endline ("[\n" ^ (allSymbols |> String.concat ",\n") ^ "\n]")
 
+let rename ~path ~line ~col ~newName =
+  let uri = Uri2.fromPath path in
+  let result =
+    match ProcessCmt.getFullFromCmt ~uri with
+    | None -> Protocol.null
+    | Some full -> (
+      let pos = Utils.protocolLineColToCmtLoc ~line ~col in
+      match References.locItemForPos ~full pos with
+      | None -> Protocol.null
+      | Some locItem ->
+        let allReferences = References.allReferencesForLocItem ~full locItem in
```

Comment:
This list I think can be non-deterministic.
Some fixed order should be chosen (e.g. by sorting the results on Uri then loc).

Reply:
I will check what's going wrong on a windows setup.

Reply:
Seems like there was an issue checking for interface file on windows

Reply:
Great thanks. I've also looked at removing some nondeterminism:
https://github.com/rescript-lang/rescript-vscode/pull/263

Which I might add later on. To make debugging across platforms easier.